v2.0/inet.md

```diff
@@ -1,6 +1,86 @@
 ---
 title: INET
-summary:
+summary: The INET data type stores an IPv4 or IPv6 address.
 toc: false
 ---
-<span class="version-tag">New in v2.0:</span> Documentation for the `INET` datatype coming soon!
+<span class="version-tag">New in v2.0:</span> The `INET` [data type](data-types.html) stores an IPv4 or IPv6 address.
+
+<div id="toc"></div>
+
+## Syntax
+
+An `INET` value can be expressed using the following formats:
+
+Format | Description
+-------|-------------
+IPv4 | Standard [RFC791](https://tools.ietf.org/html/rfc791)-specified format of 4 octets expressed individually in decimal numbers and separated by periods. Optionally, the address can be followed by a subnet mask.<br><br> Examples: `'190.0.0.0'`, `'190.0.0.0/24'`
+IPv4-mapped IPv6 | IPv6 address mapped to a IPv4 address. <br><br>Example: `'::ffff:192.168.0.1/24'`
```

Comment:
@nvanbenschoten - do you think this should this be called out as its own format? Or should it be included as an option in the IPv6 description?

Reply:
Done